src/NDItk/multigroup/AverageFissionEnergyRelease/src/ctor.hpp

```diff
@@ -0,0 +1,25 @@
+/**
+ *  @brief Default constructor for the primary group structure
+ */
+AverageFissionEnergyRelease() : RealListRecord( "fiss_q" ) {}
+
+
+/**
+ *  @brief Constructor for the primary group structure
+ *
+ *  @param[in] total       the total energy release (including delayed particles
+ *                         and neutrinos)
+ *  @param[in] prompt      the recoverable energy release (total minus delayed
```

Comment:
This is labeled "recoverable" below.

Reply:
prompt probably the better name here so I'll replace recverable